### DISCUSSION TITLE
Root Disabled By the User

### DISCUSSION THREAD

**Discussion by theoreticalphysics-cpu** [MadAnalysis/madanalysis5#235] [Q&A] [answered]
Please bear with me, because I am very new to this software. I have installed Root on Ubuntu 22.04. However, when running MadAnalysis5 in the terminal, I'm getting that Root is Disabled by the User. How can I enable it?

**Comment by BFuks** [comment, marked as answer]
Hi,

It is sufficient to udpate the file `madanalysis/input/installation_options.dat`.

Cheers,

Benjamin
